lib/resource.js

```diff
@@ -23,7 +23,7 @@ function Resource(clientFactory, actions) {
         url = url.param(params, true).query(qs).toString();
         debug('resource')(`HTTP ${config.method} ${url}`);
         debug('resource')('body:', params);
-        var req = client[method](url).body(params);
+        var req = client[method](url).body(data || params);
```

Comment:
Should we merge these objects together, instead of substituting? Is there a scenario where you need to retain certain params set, while customizing others per-request?